packages/render-fidelity-tools/src/common.ts

```diff
@@ -275,6 +281,11 @@ export class ImageComparator {
       throw new Error('Candidate image is colorless!');
     }
 
+    if (transparentPixelCount === imagePixelCount) {
```

Comment:
If the candidate image is completely transparent, it's marked as colorless ,not transparent here.  Do you want it this way or do you want it to be marked as transparent?